### DISCUSSION TITLE
cache for custom script mode.

### DISCUSSION THREAD

**Discussion by navneetankur** [davatorium/rofi#1740] [Ideas/Feature Requests]
For custom script mode, the history doesn't seem to be saved in .cache folder. Is this normal/expected? Why isn't cache available for custom script?

**Comment by DaveDavenport** [comment]
Because it to purpose dependent and it won't be possible to make a generic solution that works for a significant set for dmenu/script mode.

The script can implement a history mode itself that works for its use-case.
